src/TransactionTest.java

```diff
@@ -0,0 +1,77 @@
+import org.junit.Test;
+
+import java.util.Date;
+
+import static org.junit.Assert.*;
+
+public class TransactionTest {
+
+    @Test
+    public void givenNoDataOrMoney_returnsNoDataOrMoney() {
+        String summary = new Transaction().getSummary();
+        long money = new Transaction().getMoney();
+        String description = new Transaction().getDescription();
+        assertEquals("No data.", summary);
+        assertEquals(0, money);
+        assertNull(null,description);
```

Comment:
RED: the first `null` parameter here shouldn't be here. It should just be `assertNull(description)`